.github/workflows/test.yml

```
@@ -58,3 +58,30 @@ jobs:
       - name: Kill miden-node
         if: always()
         run: make kill-node
+
+  remote_prover_integration_tests_web_client:
+    name: remote_prover_integration_tests_web_client
```

I would name it a bit different to be more consistent with the other jobs:

```suggestion
  integration_tests_remote_prover_web_client:
    name: integration_tests_remote_prover_web_client
```